build.gradle

```diff
@@ -151,7 +151,7 @@ def versions = [
         springCloudStarterNetflixHystrix: '2.2.10.RELEASE',
         serenity                        : '2.6.0',
         serenityreporter                : '2.6.0',
-        springBootVersion               : '2.6.3',
+        springBootVersion               : '2.6.6',
```

Comment:
hi Sunil, looks like https://github.com/hmcts/probate-back-office/blob/f589a24d64c37c573b4dfe5ca41b612398c46094/build.gradle#L17 still points to 2.6.3 . Could you correct that too please.